### ISSUE TITLE
Kvantum + Forceblur => blurry outline around context menus and tooltips

### ISSUE THREAD

**Issue by Haraven** [taj-ny/kwin-effects-forceblur#89]
## Description

When I use a Kvantum theme with the forceblur effect, all context menus and tooltips have a thick, blurry outline. It doesn't seem to matter what Kvantum theme I use.

It also doesn't seem to matter whether I enable or disable menu-related blurring settings in the Kvantum Manager (as recommended in https://github.com/taj-ny/kwin-effects-forceblur/issues/70#issuecomment-2127635698).

If I don't use Kvantum, everything looks fine. If I disable forceblur, the issue goes away.

## Screenshots

![ss-11-07-2024_01-53-45](https://github.com/user-attachments/assets/275b4334-8f4d-4241-afc0-f2693d7e213d)
![ss-12-07-2024_10-18-12](https://github.com/user-attachments/assets/1d2cc441-ee55-4981-9602-12988de41f09)

## Kvantum Manager settings
![ss-12-07-2024_10-11-10](https://github.com/user-attachments/assets/747bc6e1-dc77-4190-9581-6fd1acdc6e7f)
![ss-12-07-2024_10-11-19](https://github.com/user-attachments/assets/5b25bb72-46f5-4194-b227-b95b5a67a8aa)
![ss-12-07-2024_10-11-38](https://github.com/user-attachments/assets/556b03a9-f0b1-4f4f-abab-e3ce3e392db2)
![ss-12-07-2024_10-11-32](https://github.com/user-attachments/assets/417587ec-bd28-40ca-9e51-b12ddaec8abe)

## System information
- OS: OpenSUSE Tumbleweed
- Plasmashell version: 6.1.2
- Forceblur version: latest git (b9dd76a7456e569eb3798fdf599e591ef64baf5f)
- Kvantum version: 1.1.3

**Comment by taj-ny** [comment]
Try the *develop* branch. I've also fixed an issue where menus would lose blur even on default settings.

**Comment by Haraven** [comment]
I pulled develop and rebuilt the effect (& made sure to delete `build/` beforehand). Even after restarting, the outline is still there.

**Comment by taj-ny** [comment]
The fix relies on Kvantum setting the blur region. If it doesn't do that (it does but only sometimes), the effect will use the window geometry instead, which includes the shadows. Same issue as #17.

You can either disable *Blur menus* in force blur settings or enable *Shadowless menus and tooltips* in Kvantum.

**Comment by Haraven** [comment]
I like having blurred menus, so I opted for the shadowless menus and tooltips option. **N.b.**: I had to restart my system for the Kvantum changes to take effect.

After looking into https://github.com/taj-ny/kwin-effects-forceblur/issues/17, it looks like that issue is still present, since it's not related to Kvantum. So, while KDE context menus now work fine, GTK context menus don't. I either have to live with this or disable menu blur.

I'll live with it.

Thank you for your help!
